test/unit/lib/gpHelper.js

```diff
@@ -98,4 +98,42 @@ describe('gp counts helper', () => {
       expect(message[0]).to.equal('1 GP');
     });
   });
+
+  describe('getPersonSingular', () => {
+    it('should return \'are\' for more than one male and female gp', () => {
+      const gpCounts = {
+        male: 3,
+        female: 2
+      };
+      const personSingular = gpHelper.getPersonSingular(gpCounts);
+      expect(personSingular).to.equal('are');
+    });
+
+    it('should return \'is\' for only one male gp', () => {
```

Comment:
Can this be reworded to `should return 'is' for only one male GP and zero female GPs`? This makes it clear that the zero female GPs is important.